### PR TITLE
[Storage] Use new upload server for uploads

### PR DESCRIPTION
**Pull request by adam-maj**

### CODE REVIEW

File: packages/storage/src/common/urls.ts [hunk 1]
```diff
@@ -25,6 +25,12 @@ export const DEFAULT_GATEWAY_URLS: GatewayUrls = {
  */
 export const TW_IPFS_SERVER_URL = "https://upload.nftlabs.co";
 
+/**
+ * @internal
+ */
+export const TW_UPLOAD_SERVER_URL =
+  "https://storage-server-babv-am-pinata.zeet-nftlabs.zeet.app";
```
Comment (joaquim-verges) [inline packages/storage/src/common/urls.ts:32]: Reminder to update url before merging to main

File: packages/storage/src/common/urls.ts [hunk 2]
```diff
@@ -25,6 +25,11 @@ export const DEFAULT_GATEWAY_URLS: GatewayUrls = {
  */
 export const TW_IPFS_SERVER_URL = "https://upload.nftlabs.co";
```
Comment (iketw) [inline packages/storage/src/common/urls.ts:26]: Can we then remove this const and the PINATA_IPFS_URL below?

Comment (adam-maj) [reply packages/storage/src/common/urls.ts:26]: Not sure if this is used externally is the reason I didn't use it. I have a vague memory of some other places using it outside the storage SDK (unsure if it's accurate).

Comment (dalmendray) [reply packages/storage/src/common/urls.ts:26]: Just searching in the whole repo should let you know right? I don't think we use it anywhere else

**Pull request merged**